exporter/awscloudwatchlogsexporter/factory.go

```diff
@@ -31,7 +31,8 @@ func createDefaultConfig() component.Config {
 		RetrySettings:      exporterhelper.NewDefaultRetrySettings(),
 		AWSSessionSettings: awsutil.CreateDefaultSessionConfig(),
 		QueueSettings: QueueSettings{
-			QueueSize: exporterhelper.NewDefaultQueueSettings().QueueSize,
+			QueueSize:    exporterhelper.NewDefaultQueueSettings().QueueSize,
+			NumConsumers: 1,
 		},
```

Comment:
to stay backwards compatible?

Reply:
Is this backwards compatibility or maintaining the previous default behavior?

Reply:
All uses so far don't set this value, because previously it was not possible to set. I would prefer them to not be surprised by extra memory/cpu usage coming from the fact that they are now running more consumers in parallel.